src/core/src/CoreMain.py

```diff
@@ -53,23 +53,27 @@ def __init__(self, argv):
             patch_operation_requested = execution_config.operation.lower()
             patch_assessor = container.get('patch_assessor')
             package_manager = container.get('package_manager')
+            configure_patching = container.get('configure_patching')
 
-            # if this is an auto patching installation request, log and disable (if enabled by default) the current auto OS update status. NOTE: log status in a separate file in config settings
-            if execution_config.maintenance_run_id is not None and patch_operation_requested == Constants.INSTALLATION.lower():
-                package_manager.disable_auto_os_update()
+            # # if this is an auto patching installation request, log and disable (if enabled by default) the current auto OS update status. NOTE: log status in a separate file in config settings
+            # if execution_config.maintenance_run_id is not None and patch_operation_requested == Constants.INSTALLATION.lower():
+            #     package_manager.disable_auto_os_update()
```

Comment:
Configure patching calls may not always get to the machine prior to install patches coming through. So this code needs to still remain, in the sense that if it's not disabled and an autopatching request is what's coming through, disable it.

Reply:
Also, the summary must always be written out even when it's an assessment or installation call..

Reply:
Modified the code as per this and the next comment's instructions